src/evaluations/evaluations.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Post, Put, Delete, Param, Body, UseGuards } from '@nestjs/common';
+import { EvaluationDto } from './dto/evaluation.dto';
+import { CreateEvaluationDto } from './dto/create-evaluation.dto';
+import { UpdateEvaluationDto } from './dto/update-evaluation.dto';
+import { EvaluationsService } from './evaluations.service';
+import { JwtAuthGuard } from '../guards/jwt-auth.guard';
+
+@Controller('evaluations')
+export class EvaluationsController {
+  constructor(private evaluationsService: EvaluationsService) {}
+  @UseGuards(JwtAuthGuard)
+  @Get(':id')
+  async findById(@Param('id') id: number): Promise<EvaluationDto> {
+    return this.evaluationsService.findById(id);
+  }
+
+  @UseGuards(JwtAuthGuard)
+  @Get(':id')
+  async findAll(@Param('id') id: number): Promise<EvaluationDto> {
+    return this.evaluationsService.findAll(id);
+  }
+
+  @Post()
+  async create(@Body() createEvaluationDto: CreateEvaluationDto): Promise<EvaluationDto> {
+    return this.evaluationsService.create(createEvaluationDto);
+  }
+
+  @UseGuards(JwtAuthGuard)
+  @Put(':id')
+  async update(@Param('id') id: number, @Body() updateEvaluationDto: UpdateEvaluationDto) {
```

Comment:
Please specify a return type for this function.

Reply:
Would this just be ": Promise<UpdateEvaluationDto>"

Reply:
No because you're not passing any type to Promise